### ISSUE TITLE
Is there a way to use apa instead och Chicago in Taylor & Francis template?

### ISSUE THREAD

**Issue by frankbach** [rstudio/rticles#321]
Is there a way to use apa instead och Chicago in Taylor & Francis template? Or, does a new template have to be developed?

---

By filing an issue to this repo, I promise that

- [ ] I have fully read the issue guide at https://yihui.name/issue/.
- [ ] I have provided the necessary information about my issue.
    - If I'm asking a question, I have already asked it on Stack Overflow or RStudio Community, waited for at least 24 hours, and included a link to my question there.
    - If I'm filing a bug report, I have included a minimal, self-contained, and reproducible example, and have also included `xfun::session_info('rticles')`. I have upgraded all my packages to their latest versions (e.g., R, RStudio, and R packages), and also tried the development version: `remotes::install_github('rstudio/rticles')`.
    - If I have posted the same issue elsewhere, I have also mentioned it in this issue.
- [ ] I have learned the Github Markdown syntax, and formatted my issue correctly.

I understand that my issue may be closed if I don't fulfill my promises.

**Comment by cderv** [comment]
Currently I believe this is hard coded 
https://github.com/rstudio/rticles/blob/47771dd0c64f3c352db8e0b459d1eb1fa880ccee/inst/rmarkdown/templates/tf/resources/template.tex#L75-L78

We could add a `biblio-style` variable in the pandoc template to provide in the YAML header. But was does the Taylor & Francis template advice officially  ? 

@dleutnant as you contributed the template, what is your advice on this one ? 
I believe it is based on **Taylor & Francis LATEX template for authors (Interact layout + Chicago author-date reference style)**, hence the style you put with it. 

> Or, does a new template have to be developed

@frankbach do you have a variant template of this one ? 
Some adjustment to the current one could be made easily to support several articles from taylor and francis I believe. We even have some format for two journals similar journal template with variations in the R output format. see [`joss_article`](https://github.com/rstudio/rticles/blob/3340e40084f4bb7c9a336e3943afc90bb7741d7a/R/joss_article.R#L19)

**Comment by frankbach** [comment]
I'm new to this. I hope that my answer is in the right direction.

https://www.tandfonline.com/action/authorSubmission?show=instructions&journalCode=tsed20#formatting

http://www.tandf.co.uk/journals/authors/InteractAPALaTeX.zip

Best

Frank

**Comment by cderv** [comment]
Yes it is completly relevant. It seems that the  International Journal of Science Education from taylor & francis is using a variation of the template. It is called **Taylor & Francis LATEX template for authors (Interact layout + American Psychological Association reference style** so them layout but different reference style.

From `?tf_article`, we have the link of the Tex template the format is based on: https://www.tandf.co.uk/journals/authors/InteractCADLaTeX.zip 
It seems there are just a small variation regarding reference style. 

If you are interested by **rtciles** support for **Taylor & Francis LATEX template for authors (Interact layout + American Psychological Association reference style** , you can submit a PR to suggest some adpatation. Now worry we can guide you. 

1. First I would compare the two Tex templates from the journal to see the differences. I don't think there is much

     * With Chicago : https://www.tandf.co.uk/journals/authors/InteractCADLaTeX.zip 
     * With APA: http://www.tandf.co.uk/journals/authors/InteractAPALaTeX.zip

2. Then  you can suggestion adaptation in the`tf_article()` template. 
    
    * The tex template is here: https://github.com/rstudio/rticles/blob/master/inst/rmarkdown/templates/tf/resources/template.tex 
    * And the current skeleton resources are there : https://github.com/rstudio/rticles/tree/master/inst/rmarkdown/templates/tf/skeleton

You'll find advice about submitting PR in the README: 
https://github.com/rstudio/rticles#contribute-a-new-template-format-opening-a-pull-request

Contributing a template in **rticles** is a great opportunity to practice various skills :)
You can submit the PR as draft as soon as you want, that way it is easier to help if you have question along the way, but most of all it will help the community help you on this. 

If you don't want to submit a PR that is fine, I'll just mark this issue with [the help wanted tag](https://github.com/rstudio/rticles#suggest-an-idea-for-new-format-opening-an-issue) for someone else to help. 

Thanks a lot for the idea by the way, and for pointing out that Taylor and Francis have other template !

**Comment by frankbach** [comment]
Thank you!

Since I'm new I'm afraid to do more harm than good, but it is alway interesting to learn something new. For now I would be most grateful for the help wanted tag.

Best
Frank

**Comment by cderv** [comment]
Do not hesitate to try - we are here to help and all the previous contribrution are great example of what to do. I added the label.
